src/components/Modal/Plugin.vue

```diff
@@ -0,0 +1,107 @@
+<template>
```

Comment:
We should call this component `Plugins.vue` and not `Plugin.vue` to be consistent with others components. It will conflict with the other `Plugins.vue` component, you can call that one `ProposalPlugins.vue`